.github/workflows/main.yml

```diff
@@ -1,7 +1,14 @@
-name: Build and Test
+# This worflow is triggered on push to main, pull request, or by manual invocation.
+# It builds and unit tests the codebase.
+name: Build and Unit Test
 
 on:
   workflow_dispatch:
+    inputs:
```

Comment:
I don't think this is needed given they can trigger against a PR already and they can trigger against a branch as needed.

Reply:
removed